packages/agent-toolkit/src/utils/token.utils.ts

```diff
@@ -13,9 +13,26 @@ export interface MondayTokenPayload {
   rgn: string; // region
 }
 
+/**
+ * Normalizes a token by removing common prefixes and whitespace
+ * @param token - The token to normalize (may include "Bearer " prefix)
+ * @returns The clean token string
+ */
+export const normalizeToken = (token: string): string => {
+  const trimmed = token.trim();
+
+  // Check if token starts with Bearer scheme
+  if (trimmed.toLowerCase().startsWith('bearer ')) {
+    // Split by space and return the token part (everything after "Bearer ")
+    return trimmed.split(/\s+/)[1] || trimmed;
+  }
+
+  return trimmed;
+};
+
 /**
  * Decodes a JWT token to extract the payload
- * @param token - The JWT token to decode
+ * @param token - The JWT token to decode (should be a clean token without "Bearer " prefix)
```

Comment:
It would be nice to log an error if we're unable to decode the token. It can help us find the issue faster next time.